instagram-page-react/src/pages/profilePage/organisms/posts/Posts.js

```diff
@@ -0,0 +1,59 @@
+import React, {useEffect} from 'react';
+import {useDispatch, useSelector} from "react-redux";
+
+// Lodash
+import _map from 'lodash/map';
+
+// Styles
+import styles from './posts.module.css';
+
+// Components
+import Post from "./molecules/post";
+
+// Actions
+import setPost from "./actions";
+
+// Selectors
+import {selectPosts} from "./reducers/selectors/postList.selectors";
+
+// Services
+import fetchPostsService from "./services/fetchPostsService";
+
+const renderPost = (post) => {
+    const{
+        id,
+        postPath
+    } = post;
+
+    return <Post
+        key={id}
+        postPath={postPath}
+    />;
+}
+
+function Posts() {
+    const posts = useSelector(state => selectPosts(state))
+    const dispatch = useDispatch();
+
+    useEffect( () => {
+        const fetchPosts = new Promise((resolve, reject) => {
```

Comment:
Let's maintain a loading state local to the component and set it as true initially
For the loader component, let's just show a text "Loading..." in the center of the posts section